Cargo.lock

```diff
@@ -4682,7 +4984,7 @@ dependencies = [
 [[package]]
 name = "pallet-encointer-scheduler"
 version = "1.1.0"
-source = "git+https://github.com/encointer/pallets?branch=master#f859dff21f9c9fd3863f6eaa5cf64cd1f827c7af"
+source = "git+https://github.com/encointer/pallets?branch=master#61e42c00aa5b2e2759d718b54abf4bb6a4d3199f"
```

Comment:
We seem to be inconsistent here, can you make sure that these deps also point to the polkdot-v0.9.37 branch?

Reply:
oh ok, not to master? i still dont understand our system with those version branches

Reply:
Same as the other encointer-pallet dependencies: polkadot-v0.9.37.

I think the system did change over the time. I want us to just depend on the polkadot-vx.x.x branches of our pallets. That this was different, was just an error.